jenkinsherder.py

```diff
@@ -42,21 +62,28 @@ def get_config(argv):
     parser.add_argument('--host2', required=True)
     parser.add_argument('--arch1', required=True, dest='arch1')
     parser.add_argument('--arch2', required=True, dest='arch2')
+    parser.add_argument('--os1', required=True, dest='os1')
```

Comment:
These options `os1` and `os2` are used to identify the platform with sufficient detail to meet Treeherder UI expectations. A string like 'win7 64-bit' or 'Windows 7 x64' gets converted to an identifier that shows up as 'Windows 7 x64' in Treeherder